10up/simple-podcasting
# Feature: Podcasting Image List Table

### Description of the Change

- Added the podcast image for the term into the list of terms for the taxonomy.

### Alternate Designs

None

### Benefits

- Allows admins to easily find a podcast based off of the podcast cover image
- Lets admins see the cover image more easily without editing

### Possible Drawbacks



### Verification Process

1. Create several podcast terms in the admin
2. Add a cover image to one or more
3. Confirm that the cover image shows in the list view

### Checklist:






### Applicable Issues

#32 

### Changelog Entry

#### Added
- Podcast image in the taxonomy list table view
